### DISCUSSION TITLE
hkknx findet IP Gateway nicht

### DISCUSSION THREAD

**Discussion by jhuber** [brutella/hkknx-public#220] [Q&A] [answered]
Hallo zusammen,

aktuell nutze ich Homebridge mit knxd und würde sehr gerne auf hkknx umsteigen. Ich betreibe ein Weinzierl 731 (https://weinzierl.de/de/produkte/knx-ip-interface-731/) mit einer festen IP, an einer Unify Dream Machine mit Multicast und 2 Switches. 

Leider findet hkknx das IP Interface nicht, egal ob über automatische Suche oder manuell über die feste IP/UDP. 

Für jegliche Tipps wäre ich dankbar!

**Comment by brutella** [comment, marked as answer]
- Welche Version von hkknx verwendest du?
- Hast du schonmal versucht eine Verbindung mit TCP aufzubauen?

**Reply by jhuber** [reply]
> * Welche Version von hkknx verwendest du?
> 
>     * Hast du schonmal versucht eine Verbindung mit TCP aufzubauen?

Danke für die schnelle Antwort!

-  [2.4.2-b2](https://github.com/brutella/hkknx-public/releases/tag/2.4.2-b2) bzw. aktueller 2.4.1 Release, beides getestet
-  Ja, TCP funktioniert auch nicht. Mich wundert es sehr, knxd hat ja keine Verbindungsprobleme.

**Reply by brutella** [reply]
Wieviel Tunnel Verbindungen unterstützt dein KNX Gateway? Sind womöglich alle Verbindungen belegt?

**Reply by jhuber** [reply]
> Wieviel Tunnel Verbindungen unterstützt dein KNX Gateway? Sind womöglich alle Verbindungen belegt?

Bis zu 5, allerdings hab ich nur 1 freigeschaltet, die 1 ist aber frei, d.h. knxd beende ich natuerlich oder eine potentielle ETS laeuft nicht

**Reply by jhuber** [reply]
Kleine Ergänzung, die ETS verbindet sich problemlos über die feste IP des Gateways und Port 3671. Gibt es einen Log bei hkknx, den ich irgendwie auslesen kann, der mir mehr Informationen gibt, woran es scheitert?

**Reply by jhuber** [reply]
finales update: ich war ne Weile vom Rechner weg und bekam nun die Meldung, dass die Verbindung geklappt hat, ohne etwas zu ändern (2.4.2-b2). Hatte irgendein Gerät also Schluckauf, trotzdem danke für die schnellen Antworten!

**Reply by ize86** [reply]
Ich habe das gleiche Problem!
Habe allerdings das ganze System mit dem Installer Zurückgesetzt und da gabs eine Verbindung. Nach dem aktualisieren der Software war keine Verbindung zum KNX Gateway möglich. Habe sowohl TCP als auch UDP getestet. An der KNX Konfiguration wurde nix geändert.
